sensor_msgs/Image image
---
FaceDetection[] face_detections
string error_msg
